app/helpers/application_helper/button/catalog_item_button.rb

```diff
@@ -1,8 +1,7 @@
 class ApplicationHelper::Button::CatalogItemButton < ApplicationHelper::Button::Basic
-
   def role_allows_feature?
     # when user has the privilege to edit Catalog Item, he can also add/edit buttons/button groups
-    @view_context.current_user.role_allows_any?(:identifiers => %w[catalogitem_new catalogitem_edit atomic_catalogitem_new atomic_catalogitem_edit])
+    role_allows?(:feature => %w[catalogitem_new catalogitem_edit atomic_catalogitem_new atomic_catalogitem_edit], :any => true)
```

Comment:
We could call `identifiers: []` or we could go towards a new syntax of `feature: []` (which requires the core repo merge)